source/braille.py

```diff
@@ -792,7 +792,19 @@ def getControlFieldBraille(info, field, ancestors, reportStart, formatConfig):
 		return (_("%s end") %
 			getBrailleTextForProperties(role=role))
 
-def getFormatFieldBraille(field, isAtStart, formatConfig):
+def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):

```

Comment:
> This breaks backwards compatibility because a new parameter is added in the second position.

I think this is okay. We don't call this function anywhere else and I couldn't find any matches grepping through all the add-ons hosted on the official repository. (This isn't full coverage, but I think it's highly unlikely that anyone ever calls this outside of braille itself.)